src/PsTool/XHTMLProcess.cs

```diff
@@ -505,6 +505,10 @@ protected string GetTagInfo()
 					else if (_reader.Name == "lang")
 					{
 						_lang = _reader.Value;
+						if (_lang == "utf-8")
+						{
+							_lang = string.Empty;

```

Comment:
line 513 will add this empty string to the class. I am not sure that is what we want.